### PR TITLE
media: Add stream register function

### PR DESCRIPTION
**Pull request by junmin-kim**
Add registerStream, unregisterStream, isStreamReady
Remove setStream, getStream

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9fcf82e801b159df7ae3e5563c232ac65e585348] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9fcf82e801b159df7ae3e5563c232ac65e585348] - Code Rule Check Result:
framework/include/media/InputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:146: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:146: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:146: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:153: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:153: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:153: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:163: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:164: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:76: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:146: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [20a18f111b8eb3435b4a1277edad6107dfa6cea8] - Code Rule Check Result:
framework/include/media/InputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:147: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:147: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:147: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:158: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:158: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:158: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:164: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:165: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:76: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:146: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [20a18f111b8eb3435b4a1277edad6107dfa6cea8] - Code Rule Check (C++) OK.

File: framework/src/media/BufferInputDataSource.cpp [hunk 1]
```diff
@@ -81,13 +77,14 @@ bool BufferInputDataSource::open()
 bool BufferInputDataSource::close()
 {
 	unregisterDecoder();
+	unregisterStream();
```
Comment (zhouxinhe) [inline framework/src/media/BufferInputDataSource.cpp]: I think, stop() firstly, and then unregisterStream().

Comment (junmin-kim) [reply framework/src/media/BufferInputDataSource.cpp]: Thanks for your review. I fixed it.

**Comment by seinfra** [comment]
Target : [78bf0df2e28bbc52a2f06c3170c065357179a3d6] - Code Rule Check Result:
framework/include/media/InputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:142: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:147: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:147: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:147: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:154: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:155: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:156: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:157: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:158: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:158: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:158: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:164: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:165: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:76: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:103: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:120: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:121: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:125: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:128: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:136: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:137: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:138: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:139: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:140: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:145: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:146: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [78bf0df2e28bbc52a2f06c3170c065357179a3d6] - Code Rule Check (C++) OK.

**Comment by junmin-kim** [comment]
Please do not merge.

**Pull request closed without merging**